prov/efa/src/efa.h

```diff
@@ -90,6 +90,7 @@
 #define EFA_MR_IOV_LIMIT 1
 #define EFA_MR_SUPPORTED_PERMISSIONS (FI_SEND | FI_RECV | FI_REMOTE_READ)
 
+#define EFA_RNR_INFINITE_RETRY 7
```

Comment:
trailing empty line after this line.

A comment explaining the constant would be nice to the next person.  Maybe even talking about all the valid values.

Reply:
Added a comment here. Also a comment in rxr.h around `rnr_retry` to explain all possible values.

Reply:
Added in new revision.